src/main/java/by/itechart/api/controller/UserController.java

```diff
@@ -1,32 +1,50 @@
 package by.itechart.api.controller;
 
 import by.itechart.api.entity.User;
+import by.itechart.api.service.impl.UserServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
-
+import org.springframework.web.bind.annotation.*;
 import java.util.List;
 
 @RestController
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 @RequestMapping("/users")
 public class UserController {
 
+    private final UserServiceImpl userService;
+
     @GetMapping("/all")
     public ResponseEntity<List<User>> getAllUsers() {
-        //TODO return users, not just status
-        return new ResponseEntity<>(HttpStatus.OK);
+        return new ResponseEntity<>(userService.findAll(), HttpStatus.OK);
     }
 
     /**
      * status FOUND here is just for test. It will be changed later
      * this method will get a authentication parameter later, but for now there will be no params
+     *
      * @return current user
      */
     @GetMapping("/current")
     public ResponseEntity<User> getCurrentUser() {
-        //TODO return current user, not just status
         return new ResponseEntity<>(HttpStatus.FOUND);
     }
+
+    @PatchMapping
```

Comment:
I decided to use a patch rather than put because I read a lot of articles about situations when to use a patch method and a put method. So, as we know a put method is used when we change all information in our object and I thought that this happens not quite often. But if you want - I will change it :)

Reply:
Could you please share these articles?

Reply:
> So, as we know a put method is used when we change all information in our object and I thought that this happens not quite often

That's a very rare case when we updating all fields in an object since usually there are some immutable fields in an entity in most of the cases. So, from what you are saying we should use `PUT` almost never

Reply:
> Could you please share these articles?

Sure :)